src/jalv.c

```diff
@@ -1188,5 +1232,20 @@ int
 main(int argc, char** argv)
 {
 	Jalv jalv;
-	return jalv_main(&jalv, argc, argv);
+	memset(&jalv, '\0', sizeof(Jalv));
+
+	if (jalv_open(&jalv, argc, argv)) {
+		return EXIT_FAILURE;
+	}
+
+	/* Set up signal handlers */
+	setup_signals(&jalv);
+
+	/* Run UI (or prompt at console) */
+	jalv_open_ui(&jalv);
+
+	/* Wait for finish signal from UI or signal handler */
+	zix_sem_wait(&jalv.done);
+
+	return jalv_close(&jalv);
```

Comment:
`jalv_close()` returns `0` and not `EXIT_SUCCESS`. I am not sure if this will always be equal.

Reply:
Doesn't matter, it's `0`.